components/team_sidebar/team_sidebar_controller.jsx

```diff
@@ -29,130 +30,37 @@ export default class TeamSidebar extends React.Component {
         }).isRequired,
     }
 
-    constructor(props) {
-        super(props);
-
-        this.state = this.getStateFromStores();
-    }
-
-    getStateFromStores =() => {
-        const teamMembers = TeamStore.getMyTeamMembers();
-        const currentTeamId = TeamStore.getCurrentId();
-
-        return {
-            teams: TeamStore.getAll(),
-            teamListings: TeamStore.getTeamListings(),
-            teamMembers,
-            currentTeamId,
-            show: teamMembers && teamMembers.length > 1,
-            isMobile: Utils.isMobile(),
-        };
-    }
-
-    componentDidMount() {
-        window.addEventListener('resize', this.handleResize);
-        TeamStore.addChangeListener(this.onChange);
-        TeamStore.addUnreadChangeListener(this.onChange);
-        this.props.actions.getTeams(0, 200);
-        this.setStyles();
-    }
-
-    componentWillUnmount() {
-        window.removeEventListener('resize', this.handleResize);
-        TeamStore.removeChangeListener(this.onChange);
-        TeamStore.removeUnreadChangeListener(this.onChange);
-    }
-
-    componentDidUpdate(prevProps, prevState) {
-        if (!this.state.isMobile) {
-            $('.team-wrapper').perfectScrollbar();
-        }
-
-        // reset the scrollbar upon switching teams
```

Comment:
Is this behaviour preserved somehow?

Reply:
No, it didn't work before anyway. 
Also if you have to scroll the team sidebar our UI is broken anyway.